<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NO_TEAM</class>
 <widget class="QDialog" name="NO_TEAM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CricP</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pics/Logo.png</normaloff>pics/Logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>351</width>
          <height>149</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="Team_Name_Edit">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="OK_PB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="Logo">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>pics/Logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="NO_TEAM_LB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Team Name:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
